### ISSUE TITLE
Support foreign-short for listing a short acronym for the foreign version, too

### ISSUE THREAD

**Issue by cgnieder** [cgnieder/acro#136]
**[Original report](https://bitbucket.org/cgnieder/acro/issue/136) by rugk (Bitbucket: [rugk](https://bitbucket.org/rugk), GitHub: [rugk](https://github.com/rugk)).**

----------------------------------------

The exact use case, would be to include nice words such as GDPR, General Data Protection Regulation and their translated equivalent, e.g. in German in addition to their English acronym.

I’ve asked and thus explained the use case in detail on Stackexchange: [https://tex.stackexchange.com/q/507726/98645](https://tex.stackexchange.com/q/507726/98645)

Basically, I just want to have the foreign acronym also to be listed, to give some context on how it is abbreviated in that language too.

**Comment by cgnieder** [comment]
This is implemented in [version 3.0](../tree/v3.0alpha-2), albeit indirectly

**Comment by rugk** [comment]
You likely mean with `\DeclareAcroProperty{foreign-short}` or so?

**Comment by cgnieder** [comment]
See example 9 in the manual.
